bundles/persistence/org.eclipse.smarthome.persistence.mapdb/OSGI-INF/mapdb.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file may be removed completely and replaced by annotations. See my comment on MapDbPersistenceService.